helm/karpenter-nodepools/values.schema.json

```diff
@@ -102,40 +102,124 @@
                             "type": "string",
                             "title": "Expire after",
                             "description": "The duration the controller will wait before terminating a node, measured from when the node is created. Use either 'Never' or specify duration in the format `Xs`, `Xm`, `Xh`.",
-                            "pattern": "^(([0-9]+(s|m|h))+|Never)$",
+                            "examples": [
+                                "1h",
+                                "30m",
+                                "1h30m",
+                                "Never",
+                                "45s"
+                            ],
                             "default": "720h",
-                            "examples": ["1h", "30m", "1h30m", "Never", "45s"]
+                            "pattern": "^(([0-9]+(s|m|h))+|Never)$"
                         },
                         "libVolumeSizeGB": {
-                            "type": "integer"
+                            "type": "integer",
+                            "title": "Lib volume size (GB)",
+                            "description": "Size of the volume mounted at `/var/lib` on the worker nodes. This disk is shared between kubelet folder `/var/lib/kubelet` and containerd folder `/var/lib/containerd`s.",
+                            "default": 120
                         },
                         "limits": {
                             "type": "object",
+                            "title": "Nodepool limits",
+                            "description": "These limits constrains the maximum amount of resources that the NodePool can consume",
                             "properties": {
                                 "cpu": {
-                                    "type": "string"
+                                    "type": "string",
+                                    "default": "1000",
+                                    "description": "CPU limits are described with a DecimalSI value"
```

Comment:
[minor]

I wouldn't know what `DecimalSI` means, maybe link to Kubernetes docs?

Reply:
https://github.com/giantswarm/karpenter-nodepools/pull/7